### PR TITLE
plat-d02: Use LPAE, increase pager TZSRAM size to 512K and TEE_RAM to 2M

### PR DESCRIPTION
**Pull request by jforissier**
Fixes a boot error when CFG_WITH_PAGER=y:

 INFO:    TEE-CORE:
 INFO:    TEE-CORE: Pager is enabled. Hashes: 512 bytes
 INFO:    TEE-CORE: OP-TEE version: 2.4.0-136-g4ec2358 #25 Tue Jun 13 13:32:21 UTC 2017 arm
 INFO:    TEE-CORE: Shared memory address range: 50500000, 50f00000
 ERROR:   TEE-CORE: Panic at core/lib/libtomcrypt/src/tee_ltc_provider.c:500 <get_mpa_scratch_memory_pool>

Panic occurs because tee_pager_alloc() fails to allocate memory from
tee_mm_vcore. Fix this by increasing CFG_TEE_RAM_VA_SIZE from 1 to
2 MiB. This implies to enable LPAE, otherwise the TEE core panics with:

 ERROR:   TEE-CORE: Panic 'Unsupported page size in translation table' at core/arch/arm/mm/tee_pager.c:219 <set_alias_area>

Finally, CFG_CORE_TZSRAM_EMUL_SIZE has to be increased to at least
416 KiB to avoid:

   LD      out/arm-plat-d02/core/tee.elf
 /usr/bin/arm-linux-gnueabihf-ld: OP-TEE can't fit init part into available physical memory

We choose 512 KiB because a smaller value causes horrible performance.

Signed-off-by: Jerome Forissier <jerome.forissier@linaro.org>

### CODE REVIEW

File: core/arch/arm/plat-d02/conf.mk [hunk 1]
```diff
@@ -4,8 +4,7 @@ CFG_NUM_THREADS ?= 16
 CFG_CRYPTO_WITH_CE ?= y
 CFG_WITH_STACK_CANARIES ?= y
 CFG_WITH_SOFTWARE_PRNG ?= n
-# Override default size of emulated TrustZone protected SRAM, 384 kB
-CFG_CORE_TZSRAM_EMUL_SIZE ?= 393216
+CFG_CORE_TZSRAM_EMUL_SIZE ?= 524288
```
Comment (jenswi-linaro) [inline core/arch/arm/plat-d02/conf.mk]: Perhaps the default value `458752` from `core/arch/arm/arm.mk` is good enough?

Comment (jforissier) [inline core/arch/arm/plat-d02/conf.mk]: Unfortunately, no. `time xtest 4002` takes 0.2 seconds when size is 512K, vs. 8.1 seconds when size is 448K.

Comment (jenswi-linaro) [inline core/arch/arm/plat-d02/conf.mk]: OK

**Comment by jenswi-linaro** [comment]
`Reviewed-by: Jens Wiklander <jens.wiklander@linaro.org>`

**Pull request merged**